### ISSUE TITLE
sbt-scalafix github fails with java.lang.NoSuchMethodError: coursier.Cache$.fetch

### ISSUE THREAD

**Issue by MasseGuillaume** [scalacenter/scalafix#747]
my global coursier clashes with the coursier from sbt-scalafix.

```
cat ~/.sbt/1.0/plugins/plugins.sbt
addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M4")
```

```
[error] java.lang.NoSuchMethodError: coursier.Cache$.fetch(Ljava/io/File;Lcoursier/CachePolicy;Lscala/collection/Seq;Lscala/Option;Ljava/util/concurrent/ExecutorService;Lscala/Option;)Lscala/Function1;
[error]         at scalafix.internal.sbt.ScalafixJarFetcher$.fetchJars(ScalafixJarFetcher.scala:31)
[error]         at scalafix.sbt.ScalafixPlugin$$anon$1.apply(ScalafixPlugin.scala:117)
[error]         at scalafix.sbt.ScalafixPlugin$$anon$1.apply(ScalafixPlugin.scala:111)
[error]         at java.util.HashMap.computeIfAbsent(HashMap.java:1127)
[error]         at java.util.Collections$SynchronizedMap.computeIfAbsent(Collections.java:2672)
[error]         at scalafix.sbt.ScalafixPlugin$.$anonfun$globalSettings$19(ScalafixPlugin.scala:139)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:39)
[error]         at sbt.std.Transform$$anon$4.work(System.scala:66)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:263)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error]         at sbt.Execute.work(Execute.scala:272)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:263)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:174)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error]         at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
```

workarround:

downgrade my coursier to match
https://github.com/scalacenter/scalafix/blob/8490db2edf3ef9aa6291870c04e56fa37ce4537d/project/project/plugins.sbt#L2

**Comment by olafurpg** [comment]
Superseded by https://github.com/scalacenter/sbt-scalafix/issues/1

**Comment by stewSquared** [comment]
@olafurpg The link to https://github.com/scalacenter/sbt-scalafix/issues/1 is broken.

**Comment by olafurpg** [comment]
@stewSquared this issue has been fixed and will be part of the upcoming v0.6 release
